PhysicsTools/NanoAOD/python/boostedTaus_cff.py

```diff
@@ -0,0 +1,114 @@
+       rawAntiEleCat2018 = Var("tauID('againstElectronMVA6category')", int, doc="Anti-electron MVA discriminator V6 category (2018)"),
+       idAntiEle2018 = _tauId5WPMask("againstElectron%sMVA6", doc= "Anti-electron MVA discriminator V6 (2018)"),
+)
+
+boostedTauTable.variables = _boostedTauVarsBase
+
+
+boostedTausMCMatchLepTauForTable = cms.EDProducer("MCMatcher",  # cut on deltaR, deltaPt/Pt; pick best by deltaR
+    src         = boostedTauTable.src,                 # final reco collection
+    matched     = cms.InputTag("finalGenParticles"), # final mc-truth particle collection
+    mcPdgId     = cms.vint32(11,13),            # one or more PDG ID (11 = electron, 13 = muon); absolute values (see below)
+    checkCharge = cms.bool(False),              # True = require RECO and MC objects to have the same charge
+    mcStatus    = cms.vint32(),                 # PYTHIA status code (1 = stable, 2 = shower, 3 = hard scattering)
+    maxDeltaR   = cms.double(0.3),              # Minimum deltaR for the match
+    maxDPtRel   = cms.double(0.5),              # Minimum deltaPt/Pt for the match
+    resolveAmbiguities    = cms.bool(True),     # Forbid two RECO objects to match to the same GEN object
+    resolveByMatchQuality = cms.bool(True),     # False = just match input in order; True = pick lowest deltaR pair first
+)
+
+#This requires genVisTaus in taus_cff.py
+boostedTausMCMatchHadTauForTable = cms.EDProducer("MCMatcher",  # cut on deltaR, deltaPt/Pt; pick best by deltaR
+    src         = boostedTauTable.src,                 # final reco collection
+    matched     = cms.InputTag("genVisTaus"),   # generator level hadronic tau decays
+    mcPdgId     = cms.vint32(15),               # one or more PDG ID (15 = tau); absolute values (see below)
+    checkCharge = cms.bool(False),              # True = require RECO and MC objects to have the same charge
+    mcStatus    = cms.vint32(),                 # CV: no *not* require certain status code for matching (status code corresponds to decay mode for hadronic tau decays)
+    maxDeltaR   = cms.double(0.3),              # Maximum deltaR for the match
+    maxDPtRel   = cms.double(1.),               # Maximum deltaPt/Pt for the match
+    resolveAmbiguities    = cms.bool(True),     # Forbid two RECO objects to match to the same GEN object
+    resolveByMatchQuality = cms.bool(True),     # False = just match input in order; True = pick lowest deltaR pair first
+)
+
+boostedTauMCTable = cms.EDProducer("CandMCMatchTableProducer",
+    src = boostedTauTable.src,
+    mcMap = cms.InputTag("boostedTausMCMatchLepTauForTable"),
+    mcMapVisTau = cms.InputTag("boostedTausMCMatchHadTauForTable"),                         
+    objName = boostedTauTable.name,
+    objType = cms.string("Tau"),
+    branchName = cms.string("genPart"),
+    docString = cms.string("MC matching to status==2 taus"),
+)
```

Comment:
these are all copied form PhysicsTools/NanoAOD/python/taus_cff.py
the only change are the src.

will be better if we clone them w/o reimplementing

Reply:
ok, sounds reasonable. I will change that.

Reply:
With the last commit a also reuse the WPmasks from std taus.
An import that I found to be unnecessary was removed.